README.md

```diff
@@ -1,32 +1,20 @@
 
 ## MongoDB Helm Charts repositry for Kubernetes
 
-This functionality is in alpha and is subject to change. The code is provided as-is with no warranties. Alpha features are not subject to the support SLA of official GA features.
-
-# Quick Start
-
-```helm repo add mongodb https://github.com/mongodb/helm-charts```
-```helm dependency  update```
-
-In order to install Ops Manager run this command
-
-```helm upgrade opsmanager . -n opsmanager --create-namespace  -i```
-
-In order to install MongoDB DataBase:
+## Charts
 
-```helm upgrade mongodb . --set opsManager.configMap=opsmanager-configmap --set opsManager.secretRef=opsmanager-org-access-key  -n $MONGODB_NAMESPACE --create-namespace -i```
+This repository contains sample HELM charts for different MongoDB products
 
-Where `opsmanager-configmap` and `opsmanager-org-access-key` contain OpsManager connection properties
+| Charts                  | Description                                        |
+|-------------------------|----------------------------------------------------|
+| atlas-operator          | MongoDB Atlas Helm Chart. Install Operator Chart   |
+| atlas-operator-crds     | MongoDB Atlas Collection of CRD's                  |
+|-------------------------|----------------------------------------------------|
+| ent-operator            | MongoDB Enterprise Kubernetes operator. *          |
+| ent-operator-database   | Deploy MongoDB Enterprise DataBase. *              |
+| ent-operator-opsmanager | Deploy MongoDB Enterprise OpsManager. *            |
 
-Helper script could be found at ./helpers/MongoDB-deploy.sh It contains an example that automates MongoDB Deployment using mongocli
+* Charts are in Alpha version and could be used for evaluation purposes only. Please use other methods do deploy these products. 
```

Comment:
do we need this? Could scare the customers. :)

Reply:
good point